src/pages/_app.tsx

```diff
@@ -67,15 +68,19 @@ export default function App({ Component, pageProps }: AppProps) {
                 ceramic: CERAMIC_URL,
                 connectNetwork: CERAMIC_NETWORK,
                 aliases,
+                session: true
```

Comment:
[IMO]
You said that session must be implemented independently, but what does `session:true` here mean?

Reply:
In the selfID SDK, adding `sesson:true` trigger the CACAO and did-session!
https://github.com/ceramicstudio/self.id/pull/76/files#diff-c878e8908ee4997e2ad33b48b4b22b5768fbbe7a8ec0edf8db35f29af25455c1R33